docs/getting-started-guides/aws/cloudformation-template.json

```diff
@@ -113,121 +113,177 @@
         "SecurityGroups": [{"Ref": "KubernetesSecurityGroup"}],
         "UserData": { "Fn::Base64": {"Fn::Join" : ["", [
           "#cloud-config\n\n",
+          "---\n",
+          "write_files:\n",
+          "- path: /opt/bin/waiter.sh\n",
+          "  owner: root\n",
+          "  content: |\n",
+          "    #! /usr/bin/bash\n",
+          "    until curl http://127.0.0.1:4001/v2/machines; do sleep 2; done\n",
           "coreos:\n",
```

Comment:
This is only needed for the master as it's supposed to wait for ```etcd``` to be ready before writing the network config key or any other key.
It's fixed in this PR. See ```aws/cloud-configs/master.yaml```.